### PR TITLE
Add batch prerequisites to site data interface

### PR DESCRIPTION
**Pull request by Riolku**
In conjunction with DMOJ/judge-server#1013.

Here are results locally. Generator takes no arguments and always outputs the same case for convenience.

image
image

Essentially I copied all the code from generator args for the UI and made sure it works.

Although it's not an issue to merge this before the judge PR, it's rather nonsensical.

### CODE REVIEW

**Comment by Riolku** [comment]
I want to merge DMOJ/judge-server#1019 first.

**Comment by Riolku** [comment]
The field was renamed to `dependencies`, so it has been accordingly updated.

**Review by Ninjaclasher — APPROVED** [review]
LGTM

File: judge/migrations/0132_site_data_batch_prerequisites.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.27 on 2022-03-07 19:04
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0131_spectate_contests'),
```
Comment (kiritofeng) [inline judge/migrations/0132_site_data_batch_prerequisites.py:9]: This is so out of date it makes me sad :(

**Review by kiritofeng — COMMENTED** [review]
Will assume the UI changes are sane, LGTM once you rebase

File: judge/migrations/0132_site_data_batch_prerequisites.py (hunk 1 above)
Comment (Riolku) [reply judge/migrations/0132_site_data_batch_prerequisites.py]: Rebased. This is old, wow.

**Review by kiritofeng — APPROVED** [review]
LGTM

**Pull request merged**